moveit_planners/ompl/ompl_interface/src/detail/threadsafe_state_storage.cpp

```diff
@@ -61,6 +61,7 @@ moveit::core::RobotState* ompl_interface::TSStateStorage::getStateStorage() cons
   if (it == thread_states_.end())
   {
     st = new moveit::core::RobotState(start_state_);
+    st->setToDefaultValues();  // avoid uninitialized memory
```

Comment:
FYI, this line introduced a regression on moveit2 https://github.com/ros-planning/moveit2/pull/455